### ISSUE TITLE
Zombie Pigman soldier way too powerful.

### ISSUE THREAD

**Issue by ICircinusI** [pWn3d1337/Techguns2#260]
Whenever I enter the nether I am greeted by Zombie Pigman soldiers. Which I cant kill with even the most powerful stuff that I have . I cant even go back into the nether because of them. They will kill you in seconds.

**Comment by MicrolemurDev** [comment]
Yeah they need to be nerfed. I just disable nether mobs in the config.

**Comment by ICircinusI** [comment]
Where do you do that in the config?

**Comment by Mooshroom8** [comment]
Well, Me and my friends saw that the Demons and Pigmen spawned in "Hordes" so we would bring cobblestone. We would build a little base and bring basic things like a revolver since it still does a good amount of dps. Armor we don't really use but soldier armor is cheap enough.
